x/staking/types/msg.go

```diff
@@ -178,16 +178,22 @@ func (msg MsgEditValidator) ValidateBasic() sdk.Error {
 		return sdk.NewError(DefaultCodespace, CodeInvalidInput, "nil validator address")
 	}
 
+	// TODO what if the validator just wanted to change their commission rate, do they need a description
```

Comment:
no this is all just a part of the same command, they can change the description simultaneously as the commission rate... but they don't have to, the description can just be empty in the PR and nothing will be modified

Reply:
Oh yeah they'd use the `"[do-not-modify]"` thing, ok